src/styles/blocks/head-block/icon.scss

```diff
@@ -0,0 +1,7 @@
+.icon {
```

Comment:
There are no hover effects nor transitions applied. Since the task description requires making hover changes smooth, consider using CSS transitions on the hover pseudo-class to make changes on hover smooth.